app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -110,7 +110,7 @@ public void onClick(View view) {
     }
 
     private void populateMainGroup(Menu menu) {
-        menu.add(GROUP_ID, DRAWER_HOME, Menu.NONE, R.string.home_gdg).setIcon(R.drawable.ic_drawer_home_gdg);
```

Comment:
Tested on Emulator w/ API level 16.
Checked no other usage of old image, removed in next commit.